plugins/conversion.js

```diff
@@ -0,0 +1,99 @@
+const _ = require('lodash');
+
+module.exports = (function() {
+
+	let bot;
+
+	function doConversion (message, channel) {
+
+		let [command, ...args] = message.parts;
+		let measurement, unit;
+
+		if(!args.length){
+			channel.send('Please provide something to convert.');
+			return true
+		}
+
+		//The user put a space between the measurement and the units
+		if(args.length > 1){
+			measurement = args[0];
+			unit = args[1].toLowerCase();
+		} else {
+
+			//Extract the measurement and units from a single string
+			let measurements = args[0].match(/^[\d.-]+/g);
+			let units = args[0].match(/[^\d.-]+/g);
```

Comment:
yeah - maybe measurementPart/unitPart? just something to distinguish between the two at a glance.